lib/python/picongpu/picmi/simulation.py

```diff
@@ -484,22 +431,22 @@ def get_as_pypicongpu(self) -> simulation.Simulation:
         else:
             raise ValueError("runtime not specified (neither as step count nor max time)")
 
-        util.unsupported("verbose", self.verbose)
-        util.unsupported("particle shape", self.particle_shape, "linear")
-        util.unsupported("gamma boost", self.gamma_boost)
+        pypicongpu.util.unsupported("verbose", self.verbose)
+        pypicongpu.util.unsupported("particle shape", self.particle_shape, "linear")
+        pypicongpu.util.unsupported("gamma boost, use picongpu_moving_window_move_point instead", self.gamma_boost)
```

Comment:
Extension of warning/error: not part of neither LWFA example nor ionization. Please move to separate PR.

Reply:
done